zerver/lib/actions.py

```diff
@@ -5625,6 +5627,12 @@ def do_update_message_flags(
         "messages": messages,
         "all": False,
     }
+
+    if flag == "read" and operation == "remove":
+        raw_unread_data = get_raw_unread_data(user_profile, messages)
+        unread_data = aggregate_unread_data(raw_unread_data)
+        event["unread_data"] = unread_data
```

Comment:
Wasn't there a newer proposal for what to call this?

Reply:
Yes, I was gonna change that and add OpenAPI stuff today.